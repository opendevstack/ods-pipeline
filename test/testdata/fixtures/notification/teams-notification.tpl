{
  "@type": "MessageCard",
  "@context": "http://schema.org/extensions",
  "themeColor": {{if eq .OverallStatus "Succeeded"}}"237b4b"{{else}}"c4314b"{{end }},
  "summary": "{{.ODSContext.Project}} - ODS Pipeline Run {{.PipelineRunName}} finished with status {{.OverallStatus}}",
  "sections": [
    {
      "activityTitle": "ODS Pipeline Run {{.PipelineRunName}} finished with status {{.OverallStatus}}",
      "activitySubtitle": "On Project {{.ODSContext.Project}}",
      "activityImage": "https://avatars.githubusercontent.com/u/38974438?s=200&v=4",
      "facts": [
        {
          "name": "GitRef",
          "value": "{{.ODSContext.GitRef}}"
        }
      ],
      "markdown": true
    }
  ],
  "potentialAction": [
    {
      "@type": "OpenUri",
      "name": "Go to PipelineRun",
      "targets": [
        {
          "os": "default",
          "uri": "{{.PipelineRunURL}}"
        }
      ]
    }
  ]
}
