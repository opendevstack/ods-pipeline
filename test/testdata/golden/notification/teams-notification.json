{
  "@type": "MessageCard",
  "@context": "http://schema.org/extensions",
  "themeColor": "237b4b",
  "summary": "Project - ODS Pipeline Run pipelinerun-release-v0-message-deadbeef finished with status Succeeded",
  "sections": [
    {
      "activityTitle": "ODS Pipeline Run pipelinerun-release-v0-message-deadbeef finished with status Succeeded",
      "activitySubtitle": "On Project Project",
      "activityImage": "https://avatars.githubusercontent.com/u/38974438?s=200&v=4",
      "facts": [
        {
          "name": "GitRef",
          "value": "main"
        }
      ],
      "markdown": true
    }
  ],
  "potentialAction": [
    {
      "@type": "OpenUri",
      "name": "Go to PipelineRun",
      "targets": [
        {
          "os": "default",
          "uri": "https://localhost"
        }
      ]
    }
  ]
}
